app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,18 @@
+module SessionsHelper
+  def login user
+    session[:user_id] = user.id
+  end
+
+  def current_user
+    current_user || User.find_by(id: session[:user_id]) if session[:user_id]
```

Comment:
```suggestion
    current_user ||= User.find_by(id: session[:user_id]) if session[:user_id]
```

Reply:
em edit thành ||= thì lại lỗi rubocop: Useless assignment to local variable